src/main/java/pl/byczazagroda/trackexpensesappbackend/service/WalletServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package pl.byczazagroda.trackexpensesappbackend.service;
+
+import lombok.RequiredArgsConstructor;
+
+import org.springframework.stereotype.Service;
+import pl.byczazagroda.trackexpensesappbackend.model.Wallet;
+import pl.byczazagroda.trackexpensesappbackend.model.dto.CreateWalletDto;
+import pl.byczazagroda.trackexpensesappbackend.model.dto.WalletDto;
+import pl.byczazagroda.trackexpensesappbackend.repository.WalletRepository;
+
+import java.util.UUID;
+
+@Service
+@RequiredArgsConstructor
+public class WalletServiceImpl implements WalletService {
+
+  private final WalletRepository walletRepository;
+
+  @Override
+  public WalletDto createWallet(CreateWalletDto createWalletDto) {
```

Comment:
First of all, I've no idea why u've used builder here. Second thing that it was going about creating empty service, and it's implementation.